src/main/java/org/reaktivity/nukleus/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -746,7 +742,7 @@ private void doClientFanoutReplyWindow(
             long authorization,
             long initialOffset,
             KafkaFilterCondition condition,
-            KafkaAge maximumAge,
+            KafkaOffsetType maximumAge,
```

Comment:
Please rename `maximumAge` to `maximumOffset` now that we have changed the type from `KafkaAge` to `KafkaOffsetType`.